src/main/java/mate/dao/DriverDaoImpl.java

```diff
@@ -109,4 +131,13 @@ private Driver getDriver(ResultSet resultSet) throws SQLException {
         driver.setId(id);
         return driver;
     }
+
+    private Driver parseDriverFromResultSet(ResultSet resultSet) throws SQLException {
+        long driverId = resultSet.getObject("id", Long.class);
+        String driverName = resultSet.getNString("name");
+        String licenseNumber = resultSet.getNString("license_number");
+        Driver driver = new Driver(driverName, licenseNumber);
+        driver.setId(driverId);
+        return driver;
+    }
```

Comment:
why do we need this method? We already have `getDriver(ResultSet resultSet)` with the same logic